config/projects/fuzzing.yml

```
@@ -887,6 +889,7 @@ fuzzing:
     - grant:
         - generic-worker:allow-rdp:proj-fuzzing/*
         - queue:cancel-task:fuzzing/*
+        - queue:seal-task-group:fuzzing/*
```

```suggestion
        - queue:cancel-task-group:fuzzing/*
        - queue:seal-task-group:fuzzing/*
```
Let's also include cancel task group